app/Services/Admin/SettingService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Diagnostics;
+using backend.Database;
+using backend.Dto.Responses;
+using backend.Exceptions;
+
+namespace backend.Services.Admin
+{
+    public class SettingService
+    {
+        private readonly DataContext dataContext;
+
+        public SettingService(DataContext dataContext)
+        {
+            this.dataContext = dataContext;
+        }
+
+        private string ToCamelCase(string text)
+        {
+            return System.Text.Json.JsonNamingPolicy.CamelCase.ConvertName(text);
+        }
+
+        public async Task<Dictionary<String, String>> Get()
+        {
+            var settings = this.dataContext.Settings;
+            var responseDictionary = new Dictionary<String, String>();
+            
+            foreach (var setting in settings)
+                responseDictionary.Add(
+                    this.ToCamelCase(setting.Type.ToString()), 
```

Comment:
czy tu nie będzie exception jak w settingsach zdnajdą się dwa klucze różniące się wielkością liter? dupa i DupA i wtedy będziemy dodawać do słownika pary (dupa, coś) i (dupa, coś innego) - widzę, że to jest na dole załatwione, ale mógłbym Cię prosić, żebyś dodał komentarz, że wgl nie będzie takiej sytuacji?

Reply:
nie będzie dwóch takich samych kluczy o ile developer nie doda źle dwóch ustawień z pokrywajacymi sie nazwami enuma do bazy